domain/src/test/java/org/oppia/android/domain/topic/TopicListControllerTest.kt

```diff
@@ -669,11 +635,7 @@ class TopicListControllerTest {
   }
 
   @Test
-  fun testStoryList_markLessons_progressSaved_notSaved_startedNotDone_ongoingListCorrect() {
-    storyProgressTestHelper.markStartedNotCompletedFractionsStory0(
-      profileId0,
-      timestampOlderThanOneWeek = false
-    )
+  fun testStoryList_markLessons_inProgressSaved_InProgressNotSaved_ongoingListCorrect() {
```

Comment:
Is this test/test name correct? It seems contradictory (& has a capitalization typo).

Reply:
this test is testing the ongoing list when one chapter is marked as in progress saved and another is marked as in progress not saved. I have updated the test name to `fun testStoryList_markLessonInProgressSaved_anotherLessonInProgressNotSaved_ongoingListCorrect()` to clearly indicate this.

Reply:
Ah, that's much clearer--thanks!